pkgs/stdenv/generic/check-meta.nix

```diff
@@ -710,14 +712,60 @@ let
                   cpe = makeCPE guessedParts;
                 }
               ) possibleCPEPartsFuns;
+
+          evaluateSrc =
+            config.derivationPURLInheritance && !isMarkedBroken attrs && !hasUnsupportedPlatform attrs;
```

Comment:
This is another attribute lookup that could be done globally.

Reply:
`!isMarkedBroken attrs && !hasUnsupportedPlatform attrs` is derivation specific

`config.derivationPURLInheritance` what is wrong here, in order to access the value? If we put this in the header part, it is still a lookup?